src/assets/CurrencyRateController.ts

```diff
@@ -52,10 +59,11 @@ export class CurrencyRateController extends BaseController<CurrencyRateConfig, C
     return state && state.currentCurrency ? state.currentCurrency : 'usd';
   }
 
-  private getPricingURL(currentCurrency: string, nativeCurrency: string) {
+  private getPricingURL(currentCurrency: string, nativeCurrency: string, includeUSDRate?: boolean) {
     return (
       `https://min-api.cryptocompare.com/data/price?fsym=` +
-      `${nativeCurrency.toUpperCase()}&tsyms=${currentCurrency.toUpperCase()}`
+      `${nativeCurrency.toUpperCase()}&tsyms=${currentCurrency.toUpperCase()}` +
+      `${includeUSDRate ? ',USD' : ''}`
```

Comment:
I'd suggest that we omit this if it's redundant:

```suggestion
      `${includeUSDRate && currentCurrency.toUpperCase() !== 'USD' ? ',USD' : ''}`
```

The API works the same either way at the moment, but :shrug: Why rely on that behaviour to stay the same if we can avoid it.